src/pybind/mgr/dashboard_v2/tools.py

```diff
@@ -86,10 +86,65 @@ class BaseController(object):
     Base class for all controllers providing API endpoints.
     """
     _cp_config_default = {
-        'request.error_page': {'default': json_error_page},
+        'request.error_page': {'default': json_error_page}
```

Comment:
No need to remove the comma here. Actually, I'd recommend to add that comma on the last line per default to remove potential conflicts and slightly clean-up the diff.

Reply:
agreed